docs/copied-from-beats/security/securing-beats.asciidoc

```diff
@@ -45,19 +45,15 @@ For more information about {security}, see
 {xpack-ref}/elasticsearch-security.html[Securing the {stack}].
```

Comment:
[Link](https://www.elastic.co/guide/en/apm/server/6.5/securing-beats.html). If attributes stay the same, there are no changes in this file other than grammar